instrumentation/jetty-8/src/main/java/io/opentelemetry/auto/instrumentation/jetty8/JettyHandlerAdvice.java

```diff
@@ -80,9 +83,9 @@ public static void stopSpan(
       if (!req.isAsyncStarted() && activated.compareAndSet(false, true)) {
         DECORATE.onResponse(span, resp);
         DECORATE.beforeFinish(span);
-        span.finish(); // Finish the span manually since finishSpanOnClose was false
+        span.end(); // Finish the span manually since finishSpanOnClose was false
       }
     }
-    scope.close();
+    spanScopePair.getScope().close();
```

Comment:
once we have a real `SpanWithScope`, it makes sense to add a `close()` method to it, I think.

Reply:
i hesitated to add `close()` because people might think that method applies to both the span and scope (ending the span in addition to closing the scope).

and there's no need to implement `Closeable`, since this class isn't used in try-with-resource, at least in auto instrumentation (where this class is needed/used to pass the values from one method to another), though i could see contrib instrumentation wanting to use it in try-with-resource...

Reply:
yeah, maybe a "closeScope" would be more clear.

Reply:
i like that, then don't need `getScope()`, i'll add that note to #114